### PR TITLE
[10.0] [ADD] analytic_plan_mass_create

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
migrated analytic_plan_mass_create module v10

@aheficent

### CODE REVIEW

File: analytic_plan_mass_create/wizard/analytic_plan_mass_create.py [hunk 1]
```diff
@@ -1,113 +1,98 @@
...
-        if am and 'value' in am and 'amount' in am['value']:
-            amount = am['value']['amount']
+    @api.multi
+    def _prepare_analytic_line_plan(self, wizard, item, product,
+                                    amount_currency, type, common):
+        analytic_line_plan_obj = self.env['account.analytic.line.plan']
+        line_ids = analytic_line_plan_obj.\
+            search([('account_id', '=', item.account_id.id),
+                    ('version_id', '=', wizard.template_id.version_id.id)])
+        line_ids[0].on_change_amount_currency()
```
Comment (AaronHForgeFlow) [inline analytic_plan_mass_create/wizard/analytic_plan_mass_create.py]: Got an error here when trying to create the planning lines

image

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

File: analytic_plan_mass_create/static/description/index.html [hunk 2]
```diff
@@ -7,59 +7,57 @@ <h3 class="oe_slogan">Manage the planned costs and revenues a of your analytic a
         <div class="oe_span6">
```
Comment (AaronHForgeFlow) [inline analytic_plan_mass_create/static/description/index.html]: I think this file can be removed, create a readme instead.

**Review by AaronHForgeFlow — APPROVED** [review]
Just one minor comment.

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: thanks for attending the comments. Can you squash the commits please?

**Comment by nikul-serpentcs** [comment]
@aheficent Done squashed all commits.

**Pull request merged**